packages/webapp/src/hooks/api/useCurrentUser/index.ts

```diff
@@ -59,10 +64,12 @@ export function useCurrentUser(): useCurrentUserReturn {
 			// User State
 			currentUser: filteredCurrentUser,
 			setCurrentUser,
+			updateUserPreferences,
 			userId: currentUser?.id,
 			orgId: orgId || '',
 			role: currentRole,
-			isAdmin: isAdmin
+			isAdmin: isAdmin,
+			preferences: currentUser?.preferences || {}
```

Comment:
I seem to be having an issue here, when I login to the same user in an Incognito window, the `currentUser` state does not contain the `preferences` that have been updated.  I've even tried calling the `load` fn that is exported above, but still, no luck.  Would love some feedback here, thank you.